docs/Configuration.md

```diff
@@ -38,9 +38,12 @@
 ### ApplePay component
 
```

Comment:
Overall comment: can we switch the format of parameters and values to keep it consistent with other Adyen documents? 

So, `allowOnboarding` in monospace font and **true**/**false** in bold.

Reply:
Fixed in [9532e80](https://github.com/Adyen/adyen-react-native/pull/297/commits/9532e80dd10c9f43f8b43db67dbe84b52fc2a5d8)